eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/MutableMultimap.java

```diff
@@ -80,4 +80,20 @@
 
     @Override
     <V2> MutableMultimap<K, V2> collectValues(Function<? super V, ? extends V2> function);
+
+    /**
+     * Returns a synchronized (thread-safe) collection backed by this collection.  In order to guarantee serial access,
+     * it is critical that <strong>all</strong> access to the backing collection is accomplished through the returned
```

Comment:
Use the word `multimap` instead of `collection` throughout. These types don't extend `java.util.Collection`.